### PR TITLE
Feat add action confirm/cancel modal

### PR DESCRIPTION
**Pull request by Chiranjibichapagain**
m1
m2

### CODE REVIEW

**Comment by Rostislav013** [comment]
On first screen I think should be a space between buttons.

**Review by Rostislav013 — APPROVED** [review]

File: client/src/components/Event/Event.scss [hunk 1]
```diff
@@ -5,6 +5,24 @@
...
+    font-size: $sub-head;
+    text-align: center;
+  }
+
+  &__modal-buttons {
+    margin-top: 30px;
+    width: 70%;
+    @include flexCenter;
+  }
+
```
Comment (MikeVedsted) [inline client/src/components/Event/Event.scss:25]: I think this is a good occasion to add a mixin for displaying two buttons inline, so we don't have to keep repeating this.

Please remove the class `&__modal` (and the classname for the div) and change the other two to:

```
&__modal-title {
  color: $secondary;
  font-family: $montserrat;
  font-size: $sub-head;
  text-align: center;
}

&__modal-buttons {
  @include twoButtonsInline;
  margin-top: 30px;
}
```

in the scss config, add at the bottom of the file: 

```
@mixin twoButtonsInline {
  width: 100%;
  display: flex;
  justify-content: space-between;

  & * {
    flex-grow: 0;
    width: 49%;
  }
}
```

**Review by MikeVedsted — APPROVED** [review]

**Review by MikeVedsted — APPROVED** [review]
Not sure when it was re-introduced but we need to hide the scrollbar for the modal agian.

File: client/src/components/Event/index.tsx [hunk 2]
```diff
@@ -57,11 +60,46 @@ const Event = ({ event, creatorName, participants }: EventProps) => {
...
   return (
     <div className='event'>
+      {isModalOpen && (
+        <Modal
+          closeModal={() => setIsModalOpen(false)}
+          content={
+            <div className='event__modal'>
+              <h1 className='event__modal-title'>
+                {' '}
+                {`Are you sure you want to delete the event : '${title}'?`}
```
[the remark this answers is not in the document]
Comment (Rostislav013) [reply client/src/components/Event/index.tsx:82]: Line 82 can be just `Are you sure you want to delete the event: {title}?` .

File: client/src/components/Button/Button.scss [hunk 3]
```diff
@@ -1,6 +1,7 @@
 @import '../../SCSS/config';
 
 .button {
+  width: 100%;
   font-family: $roboto;
   font-size: $text;
```
[the remark this answers is not in the document]
Comment (Chiranjibichapagain) [reply client/src/components/Button/Button.scss:6]: ye. was testing it. forgot to delete.

**Pull request merged**